packages/ui-components/rollup.config.js

```diff
@@ -30,15 +29,22 @@ const postcssPlugins = [
   autoprefixer(),
 ];
 
-const config = outputs.map(({file, format}) => ({
+const config = outputs.map(({ file, format }) => ({
   input: 'src/lib/index.js',
   output: {
     file,
     format,
-    name: 'TextEditor',
+    name: 'CiceroUI',
```

Comment:
Confused by this change. Why `CiceroUi` ?

Reply:
I must have had a fugue state when i made this change 🤷‍♀️